### DISCUSSION TITLE
Show TOC sidebar in publicly shared view

### DISCUSSION THREAD

**Discussion by wohenbushuang** [outline/outline#3599] [Self Hosting] [answered]
I seem to remember that before 0.64 there is a TOC toggle.

**Comment by tommoor** [comment, marked as answer]
Make sure you're running 0.64.3 - a bug was fixed

**Reply by wohenbushuang** [reply]
yep im running 0.64.2

**Reply by tommoor** [reply]
Ah, sorry I meant 0.64.3 – edited.

**Reply by wohenbushuang** [reply]
Confirm fixed in 0.64.3. Great!
BTW, request it displays by default on large screens.

**Reply by tommoor** [reply]
Ah yes, maybe. It will stay on once toggled though 👍
